source/MRViewer/MRRibbonNotification.cpp

```diff
@@ -262,6 +262,7 @@ void RibbonNotifier::drawNotifications_( float scaling, float scenePosX )
     const float width = 337.0f * scaling;
     currentPos.x += padding.y;
 
+    int filteredNumber = -1;
```

Comment:
does this number change?

Reply:
it is necessary to remember the closed notification and close it (after the draw is completed)
line 415 and 462